components/Tabs2/src/private/MDCTabBarViewItemView.m

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCTabBarViewItemView.h"
+
+/** The minimum width of any item view. */
+static const CGFloat kMinWidth = 90;
+
+/** The maximum width of any item view. */
+static const CGFloat kMaxWidth = 360;
+
+/** The minimum height of any item view. */
+static const CGFloat kMinHeight = 48;
+
+@implementation MDCTabBarViewItemView
+
+- (CGSize)intrinsicContentSize {
+  return CGSizeMake(kMinWidth, kMinHeight);
+}
+
+- (CGSize)sizeThatFits:(CGSize)size {
+  return CGSizeMake(MIN(kMaxWidth, MAX(kMinWidth, size.width)), MAX(kMinHeight, size.height));
```

Comment:
This line is a little difficult to parse – maybe break this out into `width` and `height` variables

Reply:
clang-format won't let me. ;_;

Reply:
Split into two autos.  Please take a look.